.github/workflows/rails_releaser_tests.yml

```
@@ -0,0 +1,27 @@
+name: Rails releaser tests
+
+on:
+  pull_request:
+    paths:
+    - "tools/releaser/**"
+  push:
+    paths:
+    - "tools/releaser/**"
+
+permissions:
+  contents: read
+
+jobs:
+  releaser_tests:
+    name: releaser tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Remove Gemfile.lock
+        run: rm -f Gemfile.lock
+      - name: Set up Ruby 3.3
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.3
```

Do you think we can roll with `ruby-version: ruby`? This will just grab the latest stable version of Ruby (per [documentation](https://github.com/ruby/setup-ruby?tab=readme-ov-file#supported-version-syntax)) and give us one less thing to maintain.

```suggestion
      - name: Set up Ruby
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: ruby
```

---

I think it might be worth revisiting, we've had some trouble in the past where gems from the oldest Rails version we're maintaining are broken against the most recent Ruby version, but it's probably best to air on the side of most recent and revisit when it surfaces as a problem. 👍